themes/edx.org/lms/templates/header.html

```diff
@@ -69,10 +69,9 @@ <h2 class="course-header">
 
     % if user.is_authenticated():
       % if not course or disable_courseware_header:
-        % if not nav_hidden or show_program_listing:
           <nav aria-label="Main" class="nav-main">
             <ul class="left list-inline nav-global authenticated">
-              % if not nav_hidden:
+              % if not nav_hidden and not show_program_listing:
```

Comment:
@schenedx [Line](https://github.com/edx/edx-platform/pull/12469/files#diff-26f71f23dbccbf70d6b2918e39ba566fL88) If `show_programs_listing` is `True` we will show tabs view. While both are `True` first condition will return `False` and second condition will return `True` because of `show_programs_listing` is `True`.